exonum/src/runtime/blockchain_data.rs

```diff
@@ -132,3 +132,18 @@ impl SnapshotExt for dyn Snapshot {
         mount_point_for_service(self, id)
     }
 }
+
+/// Extension trait for `Fork` allowing to writeable access blockchain data in a more structured way.
+///
+/// Be very careful with blockchain manipulations, they can destroy the data consistency!
+#[doc(hidden)]
+pub trait ForkExt {
```

Comment:
Nit: IMO, exposing a constructor for `blockchain::Schema` (with `#[doc(hidden)]` and maybe feature gate) could be more versatile.